pubspec.lock

```diff
@@ -1091,5 +1091,5 @@ packages:
     source: hosted
     version: "3.1.3"
 sdks:
-  dart: ">=3.7.0 <4.0.0"
+  dart: ">=3.6.0 <4.0.0"
```

Comment:
@vlacko0930 ez alapján olyan, mintha kicsit régebbi Flutter verzió lenne nálad (3.29-nél régebbi) és azért kellett a többi dependency-hez is hozzányúlni. A gitignore módosítások jöhetnek, de a dependency-ket dependabottal frissítjük és emiatt most van egy kis conflict, ezt jó lenne feloldani, hogy mehessen a merge.